### DISCUSSION TITLE
Modifica interfaccia supplenze

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#263] [Idee e proposte]
Salve, sarebbe possibile inserire un'interfaccia simile a quella utilizzata in axios per quanto riguarda la sezione supplenze?
Un docente della classe non riesce a supplire altrimenti. Intendo inserire un menù a tendina dove scegliere docente da supplire e materia/classe.

**Comment by trinko** [comment]
La segno come richiesta.
